pkg/action/lint.go

```diff
@@ -0,0 +1,112 @@
+/*
+Copyright The Helm Authors, Suse LLC
```

Comment:
Nit: maybe SUSE should be in uppercase (we should fix the linter, because right now it's passing as it is).

Reply:
(also, other headers too).